{"name": "test_login_01[username]", "status": "broken", "statusDetails": {"message": "_pytest.fixtures.FixtureLookupError: ('login', <FixtureRequest for <Function test_login_01[username]>>)", "trace": "file /Users/test/work/API_Testframe/TestCase/test_login.py, line 35\n      @allure.feature('Login')\n      @allure.severity('blocker')\n      @allure.story('测试登录成功')\n      @pytest.mark.parametrize(\"login\", Datas().data[1]['postdata'], indirect=True)\n      def test_login_01(self, action,getdata,login):\nE       fixture 'login' not found\n>       available fixtures: action, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, getdata, metadata, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory\n>       use 'pytest --fixtures [testpath]' for help on them.\n\n/Users/test/work/API_Testframe/TestCase/test_login.py:35"}, "description": "\n            用例描述：login接口\n        ", "attachments": [{"name": "log", "source": "6736b79c-a3a3-4ac8-a615-4096090a59de-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "login", "value": "'username'"}], "start": 1587625451743, "stop": 1587625451743, "uuid": "628b76e4-fbfd-4bd2-9a5f-86effbb01d86", "historyId": "e20d489f72b6f5baf79e9fcbe3ac31fa", "testCaseId": "49d28fbbfb79bd9a490d1da4fd8fe957", "fullName": "TestCase.test_login.TestLogin#test_login_01", "labels": [{"name": "feature", "value": "Login"}, {"name": "severity", "value": "blocker"}, {"name": "story", "value": "测试登录成功"}, {"name": "parentSuite", "value": "TestCase"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Lettos-iMac.local"}, {"name": "thread", "value": "89973-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCase.test_login"}]}